app/src/main/java/org/mozilla/fenix/share/ShareController.kt

```diff
@@ -111,6 +120,7 @@ class DefaultShareController(
             when (e) {
                 is SecurityException, is ActivityNotFoundException -> {
                     snackbar.setText(context.getString(R.string.share_error_snackbar))
+                    snackbar.setLength(FenixSnackbar.LENGTH_LONG)
```

Comment:
Why make this one long?

Reply:
By default if we didn't specify length it was set to long.  So I kept this one as long.